build/site/404.html

```diff
@@ -26,13 +26,13 @@
         <div class="navbar-item has-dropdown is-hoverable">
           <div class="navbar-link">Download</div>
           <div class="navbar-dropdown">
-            <a class="navbar-item" href="/specifications/specifications/1.3/specs/_attachments/C2PA_Specification.pdf">Technical Specifications</a> 
-            <a class="navbar-item" href="/specifications/specifications/1.2/explainer/_attachments/Explainer.pdf">Explainer</a> 
-            <a class="navbar-item" href="/specifications/specifications/1.2/guidance/_attachments/Guidance.pdf">Guidance for Implementers</a> 
-            <a class="navbar-item" href="/specifications/specifications/1.2/ux/_attachments/UX_Recommendations.pdf">User Experience Guidance</a> 
+            <a class="navbar-item" href="/specifications/specifications/1.0/specs/_attachments/C2PA_Specification.pdf">Technical Specifications</a> 
```

Comment:
All these links on the 404 page are being pointed at 1.0 versions. Is that intended?